src/app.ts

```diff
@@ -127,13 +211,28 @@ export function buildApp(
 		'/signed-image-url',
 		(req: express.Request, res: express.Response) =>
 			withPandaAuth(getPanda, req, res, () => {
-				const config = { url: req.query.url };
-				handleImageSigning(
-					config as SignedImageUrlConfig,
-					getPanda,
-					req,
-					res,
-				);
+				const config: SignedImageUrlConfig = {
+					url: req.query.url as string,
+					profile: {
+						width: DEFAULT_WIDTH,
+					},
+				};
+				if (config.profile && req.query.width) {
+					config.profile.width = Number.parseInt(
+						req.query.width.toString(),
+					);
+				}
+				if (config.profile && req.query.height) {
+					config.profile.height = Number.parseInt(
+						req.query.height.toString(),
+					);
+				}
+				if (config.profile && req.query.quality) {
+					config.profile.quality = Number.parseInt(
+						req.query.quality.toString(),
+					);
+				}
```

Comment:
Oh how come the config building here changed? Did we not already support these transformation params when creating the signed image URL?

Reply:
We're not using the GET endpoint any more in the UI, does it make sense to revert this change now as this isn't needed any longer? (Supporting these params on the GET endpoint feels like a valid thing do do, but perhaps belongs in a separate PR).